docs/application/tizen-studio/native-tools/ui-builder/custom-ui-component.md

```diff
@@ -55,17 +55,17 @@ A custom component requires an EDJ resource file (`[ComponentName].edj`) and an
     | <component> | <state>     | default_value | State  property default value. The default value must be a name attribute defined in the <value> element attributes. | String  | Yes      | -                                    |
     | <state>     | <value>     | name          | Value  property identifier               | String  | Yes      | -                                    |
     | <state>     | <value>     | display_name  | Name  shown on the value property label in the Properties view | String  | No       | name attribute in the <value>element |
-    | <state>     | <value>     | signal        | EDC  signal used to change the UI state at runtime.  For information on using signals, see [Program Block](../../../org.tizen.guides/html/native/ui/efl/learn_edc_program_n.htm). | String  | Yes      | -                                    |
+    | <state>     | <value>     | signal        | EDC  signal used to change the UI state at runtime.  For information on using signals, see [Program Block](../native/guides/ui/efl/learn-edc-program.md). | String  | Yes      | -                                    |
```

Comment:
It looks `../../../native/guides/ui/efl/learn-edc-program.md` is correct.

Reply:
done